frontend/src/Components/LycheMenuDivider/LycheMenuDivider.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from './LycheMenuDivider.module.scss';
+
+interface MenuCategory {
```

Comment:
Vi pleier å bruke type
```suggestion
interface LycheMenuDividerProps {
```

Reply:
done

Reply:
Forslaget mitt glemte å endre type

Reply:
derda